src/components/CategoryCard/CategoryCard.js

```diff
@@ -1,9 +1,16 @@
+import { useQuiz } from "contexts";
 import { Link } from "react-router-dom";
 import "./CategoryCard.css";
 
 export const CategoryCard = ({ categoryName, image }) => {
+  const { quizDispatch } = useQuiz();
   return (
-    <Link to="/rules">
+    <Link
+      to="/rules"
+      onClick={() =>
+        quizDispatch({ type: "SET_CATEGORY", payload: categoryName })
```

Comment:
could use constants for types!!(just a suggestion)

Reply:
Sure. Thanks for the suggestion.